internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -44,15 +54,38 @@ func getBillingModelForCloudProvider(cloudProvider string) (amsv1.BillingModel,
 	return "", errors.InvalidBillingAccount("unsupported cloud provider")
 }
 
-func newBaseQuotaReservedResourceResourceBuilder() amsv1.ReservedResourceBuilder {
-	rr := amsv1.ReservedResourceBuilder{}
-	rr.ResourceType("cluster.aws") //cluster.aws
-	rr.BYOC(false)                 //false
-	rr.ResourceName("rhosak")      //"rhosak"
-	rr.BillingModel("marketplace") // "marketplace" or "standard"
```

Comment:
We don't set the BillingModel in this method anymore as we always end up setting it in another part of the `ReserveQuota` method